### PR TITLE
ADD: polish language support

### PR DESCRIPTION
**Pull request by InYourHead**
I added polish translations.

I didn't translated Watchface yet - I have no idea what's that for.

### CODE REVIEW

File: sources/situations/pl/application.ts [hunk 1]
```diff
@@ -0,0 +1,312 @@
...
+        <source>Ok</source>
+        <translation type="unfinished">Ok</translation>
+    </message>
+    <message id="cancel">
+        <source>Cancel</source>
+        <translation type="unfinished">Anuluj</translation>
+    </message>
+    <message id="next">
+        <source>Next</source>
+        <translation type="unfinished">Natępny</translation>
```
Comment (InYourHead) [inline sources/situations/pl/application.ts:77]: Should be "Następny"

File: sources/situations/pl/application.ts [hunk 2]
```diff
@@ -0,0 +1,312 @@
...
+        <source>Privacy policy</source>
+        <translation type="unfinished">Polityka prywatności</translation>
+    </message>
+    <message id="translation">
+        <source>Language</source>
+        <translation type="unfinished">Język</translation>
+    </message>
+    <message id="translation_updated">
+        <source>Application language updated!&#60;br&#62;&#60;br&#62;NOTE: Parts of the application may not be translated at all times. More info here:&#60;br&#62;%1</source>
+        <translation type="unfinished">Język aplikacji został zaktualizowany!&#60;br&#62;&#60;br&#62;UWAGA: Fragmeny aplikacji mogą być czasami niezaktualizowane. Więcej informacji znajdziesz tutaj:&#60;br&#62;%1</translation>
```
Comment (InYourHead) [inline sources/situations/pl/application.ts:153]: Should be "Fragmenty" instead of "Fragmeny"

File: sources/situations/pl/index.ts [hunk 3]
```diff
@@ -0,0 +1,279 @@
...
+        <source>Get all plugins!!</source>
+        <translation type="unfinished">Zdabądź wszystkie dodatki!!</translation>
+    </message>
+    <message id="plugins_description_index">
+        <source>***</source>
+        <translation>***</translation>
+    </message>
+    <message id="ambience_title_index">
+        <source>Ambience</source>
+        <translation type="unfinished">Nastrój</translation>
```
Comment (InYourHead) [inline sources/situations/pl/index.ts:16]: I am not sure of Ambience translation -> I don't know a word context

**Comment by hhaveri** [comment]
Nice work!

Watchface feature is currently only supported on Sailfish. It is there for controlling Pebble smartwatch watch face. "Watchface" basically means the clock background graphics.

"Ambience" is Sailfish OS terminology - basically like a "theme". I believe it has been translated as "atmosfera" here:
https://translate.sailfishos.org/pl/jolla-gallery-ambience/translate/gallery-ambience.ts

Can you please remove the *type="unfinished"* attributes from the ready translations?

**Comment by hhaveri** [comment]
Finally I have some spare time, so I'll merge this and also remove the "unfinished" tags.

Please send mail to support@pastillilabs.com if you would like to specify how to get credits / mentions for your work on our web page & social media & get some extra compensation!

**Comment by InYourHead** [comment]
I'm sorry, I was overhelmed at work. @hhaveri  I don't need any credits, I did it, because I have friends, who woud like to use Situations, but they don't understand English. I will check current translations and create new pull request to fix typos at the end of a week.
Have a good day!

**Pull request merged**